web/tendrl/cluster_import.rst

```diff
@@ -12,7 +13,13 @@ Import existing cluster.
 Setup
 =====
 
-#. Login user into Tendrl, see :doc:`/web/tendrl/login_positive`
+#. Prepare Gluster cluster with 2 Volumes.
+
+#. Enable Gluster Volume Profiling on one Volume:
```

Comment:
Wouldn't be bad if you linked upstream documentation for gluster volume profiling.